dts/bindings/usb/usb.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This should probably be usb-ep.yaml (or something to make it clear the props are related to being an endpoint).

Reply:
Should I split it into two files, one usb-ep.yaml (only the ep properties) and one usb.yaml (compatible, clocks, .etc.)?

Reply:
that probably makes sense